compose/storage/postgres/docker-compose.yaml

```diff
@@ -0,0 +1,45 @@
+version: '3.7'
+
+services:
+  postgres:
+    container_name: postgres_container
+    image: postgres
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:-myUsr}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-supersecretpass}
+      PGDATA: /data/postgres
+    volumes:
+       - postgres:/data/postgres
+    ports:
+      - "5432:5432"
```

Comment:
Shall we move `environment`, `volumes` and `ports` configuration inside `docker-compose.dev.yaml` file?

In that way we keep the base declaration in our `docker-compose` file, but then we can override specific settings for dev environment in  `docker-compose.dev.yaml`.